docs/maintenance/ISSUES.md

```diff
@@ -0,0 +1,101 @@
+     - If it's not an obvious duplicate, link to the existing issue and explain why
+   - If the code is working as intended:
+     - Add the `working as intended` label and remove the `bug` and `triage` labels
+     - If the behavior is due to the user doing something wrong, such as an incorrect config:
+       - Add the `fix: user error` label
+       - [This issue search has some examples of closing comments](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+sort%3Aupdated-desc+label%3A%22fix%3A+user+error%22+is%3Aclosed)
+     - If the behavior is otherwise expected, [this issue search has some examples of closing comments](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+sort%3Aupdated-desc+label%3A%22working+as+intended%22+-label%3A%22fix%3A+user+error%22+is%3Aclosed+)
+     - You needn't go into too much detail in your comment - just enough to explain it
+3. If it was valid, add the `accepting prs` label and remove the `triage` label
+4. If you know the rough steps for a fix, consider writing a comment with links to codebase to help someone put together a fix
+5. If you think that the fix is relatively straightforward then consider also adding the `good first issue` label
+
+Whenever an issue is waiting for the reporter to provide more information, it should be given the `awaiting response` label.
+When more information is provided:
+
+- If you have time to go through the triage flow again, do so
+- If you don't have time, add the `triage` label and remove the `awaiting response` label
+
+:::tip
+If your link is both a "permalink" (includes a commit hash instead of a branch name) and has a line number/line range then GitHub will embed the code in your comment.
+When viewing a file in GitHub pressing `y` will update the URL to a "permalink" with the current commit hash, then you can select the relevant lines and paste that URL into the comment.
+:::
+
+### Determining Whether Code is Working As Intended
+
+As you become more familiar with the codebase and how everything works this will be easier to do intuitively, but to begin with this will likely involve investigating the documentation, code, and the tests to determine if it's a bug or working as intended.
+In general if there is a passing test or documented example that is the same as or similar to the repro code - that indicates that it's working as intended.
+If you can't find anything that matches, use your best judgement based on the spirit of the code.
+
+### Looking for Duplicates
+
+It's worth noting that occasionally a user will intentionally raise a duplicate issue because they feel the original issue was closed when it shouldn't have been.
+If this is the case then you should read the original issue to gather context and understand the reason for it being closed, and determine if the new issue is raises any new or relevant issue that requires addressing.
```

Comment:
```suggestion
It's worth noting that, occasionally, a user will intentionally raise a duplicate issue because they feel the original issue was closed when it shouldn't have been.
If this is the case, you should read the original issue to gather context, understand the reason for it being closed, and determine if the new issue raises any new or relevant point that requires addressing.
```